src/app/containers/ATIAnalytics/canonical/index.test.jsx

```diff
@@ -29,11 +29,11 @@ describe('Canonical ATI Analytics', () => {
   });
 
   it('should render a noscript image for non-JS users', () => {
-    const renderedATI = render(
+    const { renderedATI } = render(
```

Comment:
Not sure this key is right, i think it needs to be something that `render` provides?